tests/test_other.py

```diff
@@ -2637,6 +2637,37 @@ def test_node_catch_exit(self):
     self.assertContained(reference_error_text,
                          run_js ('index.js', engine=NODE_JS, stderr=STDOUT, assert_returncode=None))
 
+  def test_extra_exported_methods(self):
+    # Test with node.js that the EXTRA_EXPORTED_RUNTIME_METHODS setting is considered by libraries
+    if NODE_JS not in JS_ENGINES:
+      return
```

Comment:
can write `return self.skip("..reason..")` (moving the explaining comment into the reason; then it's shown when skipped)

lgtm aside from that, thanks.